src/pages/TeamSettings/TeamSettings.vue

```diff
@@ -27,7 +27,11 @@ export default {
   },
   computed: {
     ...mapGetters('api', ['isCloud']),
-    ...mapGetters('tenant', ['tenant', 'role'])
+    ...mapGetters('tenant', ['tenant', 'role']),
+    ...mapGetters('license', ['hasPermission']),
+    isTenantAdmin() {
```

Comment:
This probably isn't necessary since this button will be removed soon and isn't visible anyway